src/app/model/services/administrator.service.ts

```diff
@@ -0,0 +1,33 @@
+import {HttpResponse, HttpClient} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs';
+
+import {StalkerEndpoint} from './stalker-endpoint';
+
+export interface AdminGetType {
+  email: string;
+  role: string;
```

Comment:
questo è ancora string, come mai?

Reply:
onestamente in questo territorio mi muovo alla cieca non avendo endpoint, molto probabilmente dopo andrà rifattorizzato ma ho pensato che per ogni administrator mi restituiscono la coppia email e role, poi nel component converto string all'enum